ConceptMap/ConceptMap-Genomics-primary-sample-state.json

```diff
@@ -0,0 +1,141 @@
+{
+  "resourceType": "ConceptMap",
+  "id": "genomics-primary-sample-state",
+  "url": "https://fhir.nhs.uk/ConceptMap/genomics-primary-sample-state",
+  "version": "0.0.1",
+  "name": "GenomicsPrimarySampleState",
+  "title": "Genomics Primary Sample State",
+  "status": "draft",
+  "date": "2024-08-13T00:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording the concepts on method of sample preservation. This is intended for the element Specimen.condition",
```

Comment:
This may be under Specimen.container.type depending on the type of information the MDS is specifying, needs clarification from Adam/the genomics SMEs